chrome/views/popupView.js

```diff
@@ -52,7 +54,8 @@ GmailToTrello.PopupView.prototype.init = function() {
     // inject a button & a popup
     this.confirmPopup();
 
-    setInterval(function() {
+    clearInterval(self.detectButtonInterval);
+    self.detectButtonInterval = setInterval(function() {
```

Comment:
This isn't part of the project. It's just a separate bug fix. Probably worth merging right away. It was creating multiple intervals on top of each other. This just clears one interval when it creates the next.